src/components/screens/software-registry-screen.tsx

```diff
@@ -342,6 +344,14 @@ const SoftwareScreen = () => {
         disabled={loading}
         existingSoftwareList={software}
       />
+      <Card sx={{ mt: 2, width: "100%"}}>
```

Comment:
Does this need to be in it's own card?
You could actually refactor this into it's own generic component and then use it in all the appropriate places where the back button is used. Cleans the code a little